src/plus/integrations/providers/models.ts

```diff
@@ -479,3 +495,183 @@ export function toAccount(account: ProviderAccount, provider: ProviderReference)
 		username: account.username ?? undefined,
 	};
 }
+
+export const toProviderBuildStatusState = {
+	[PullRequestStatusCheckRollupState.Success]: 'SUCCESS' as GitBuildStatusState,
+	[PullRequestStatusCheckRollupState.Failed]: 'FAILED' as GitBuildStatusState,
+	[PullRequestStatusCheckRollupState.Pending]: 'PENDING' as GitBuildStatusState,
+};
+
+export const toProviderPullRequestReviewState = {
+	[PullRequestReviewState.Approved]: 'APPROVED' as GitPullRequestReviewState,
+	[PullRequestReviewState.ChangesRequested]: 'CHANGES_REQUESTED' as GitPullRequestReviewState,
+	[PullRequestReviewState.Commented]: 'COMMENTED' as GitPullRequestReviewState,
+	[PullRequestReviewState.ReviewRequested]: 'REVIEW_REQUESTED' as GitPullRequestReviewState,
+	[PullRequestReviewState.Dismissed]: null,
+	[PullRequestReviewState.Pending]: null,
+};
+
+export const toProviderPullRequestMergeableState = {
+	[PullRequestMergeableState.Mergeable]: 'MERGEABLE' as GitPullRequestMergeableState,
+	[PullRequestMergeableState.Conflicting]: 'CONFLICTS' as GitPullRequestMergeableState,
+	[PullRequestMergeableState.Unknown]: 'UNKNOWN' as GitPullRequestMergeableState,
+};
```

Comment:
Ahh yes thanks for reminding me! When I originally wrote this, these enums were being exported as types from the shared library so I couldn't use their values. That has since been fixed, and I forgot to update these.

Done in commit: [Uses enum values](https://github.com/gitkraken/vscode-gitlens/pull/3200/commits/f89dd9bf9d93986293ffb27bcdf199efe003357a)

Nice catch!